gnd/src/main/java/com/google/android/gnd/GndApplicationModule.java

```diff
@@ -83,4 +100,73 @@ static SharedPreferences sharedPreferences(GndApplication application) {
         .getApplicationContext()
         .getSharedPreferences(SHARED_PREFERENCES_NAME, Context.MODE_PRIVATE);
   }
+
+  @Provides
+  @Singleton
+  static LocalDatabase localDatabase(Context context) {
+    return Room.databaseBuilder(context, LocalDatabase.class, Config.DB_NAME)
+        // TODO(#128): Disable before official release.
+        .fallbackToDestructiveMigration()
+        .build();
+  }
+
+  @Provides
```

Comment:
Instead of having separate Providers here, can we instead just use `@Inject` on each Dao?

Reply:
Nvm, I don't see an easy way to get the annotation to be applied to the generated DAO classes.